util.py

```diff
@@ -3,7 +3,7 @@
 from flask import make_response
 from common import DailyPrayer
 
-def EncodeParameter(param, spaced):
+def EncodeParameter(param, spaced=None):
```

Comment:
since spaced is a bool i don't think it's right to set this to None - it should be false.